packages/components/data-table/src/cell.js

```diff
@@ -17,44 +22,138 @@ import {
   SortableHeaderInner,
   RowExpandCollapseButton,
 } from './cell.styles';
+import Resizer from './column-resizer';
+
+const HeaderCellWrapper = (props) => {
+  const [colResizingState, setColResizingState] = React.useState({
+    isResizing: false,
+    initialColWidth: undefined,
+    initialMousePosition: undefined,
+  });
+  const tableRef = props.tableRef;
+  const headerRef = React.useRef(null);
+
+  const onStartResizing = (e) => {
```

Comment:
It is not a required change, but would be more clear if we could name the variable `event`